projects/mysql-server/targets/fuzz_mysqld.dict

```diff
@@ -0,0 +1 @@
+user="root"
```

Comment:
What do you mean here ?

I put this because I guess fuzzing needs it to find a right authentication sequence.

Reply:
My point was that the fuzzers would probably benefit from dictionaries like [this](https://github.com/google/AFL/blob/master/dictionaries/sql.dict) (which was written for SQLite, might need to be modified to work better for mysql)

Reply:
Can it do anything if it doesn't find the right sequence? Maybe the target function should perform the sequence regardless of the input (you know the target better than I, just a suggestion).

Reply:
Ok, maybe it will be more relevant for another more restricted target...

If it does not find the right sequence, it is still fuzzing pre authentification code

It would surely be a good idea to fuzz post authentification only, but that requires to write another fuzz target with a bit more initialization...